ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -129,7 +129,10 @@ private async Task<IEnumerable<IList<Notification>>> GetGroupedResultsAsNotifica
                 };
                 notification.SocialContextAddresses = notificationSocialContextAddresses;
                 notification.SocialContextVenues = notificationSocialContextVenues;
-                notification.TreatmentEvents = notificationTransferEvents; //TODO combine with outcome events
+                foreach (var notificationTransferEvent in notificationTransferEvents)
```

Comment:
currently death events are added in AsNotification in ExtractTreatmentEvents but were getting overwritten here. Do we want to leave the TODO comment here for future treatment events/outcomes?